### PR TITLE
update user guide

### PR DESCRIPTION
**Pull request by ziyi22**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -4,9 +4,16 @@
...
 ---
 
-# EffiTrack User Guide
+# User Guide
 
-HR departments often struggle to accurately measure employee efficiency, leading to reduced productivity. Traditional methods of tracking employee performance, such as manual documentation or outdated systems, are time-consuming and prone to errors. EffiTrack solves these challenges by providing a centralized platform to **effortlessly monitor employee efficiency**.
+## Introduction
+
+EffiTrack is a **desktop app for managing employees, tracking the efficiency of employees, 
+optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
```
Comment (howenyap) [inline docs/UserGuide.md]: 'Command Line Interface (CLI)': Command is missing
'AB3': Change to effitrack

**Review by howenyap — COMMENTED** [review]
Some typos in the UserGuide

File: docs/UserGuide.md [hunk 2]
```diff
@@ -17,6 +24,7 @@ HR departments often struggle to accurately measure employee efficiency, leading
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
+
```
Comment (yiwern5) [inline docs/UserGuide.md:27]: can omit this extra line

File: docs/UserGuide.md [hunk 3]
```diff
@@ -230,7 +254,9 @@ Action     | Format, Examples
 **Clear**  | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Edit**   | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [d/DEPARTMENT] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
+**Assign** | `assign task/TASK TITLE by/dd-MM-yyyy to/INDEX`<br> e.g., `assign task/Complete Project Proposal by/22-05-2023 2359 to/1`
```
Comment (yiwern5) [inline docs/UserGuide.md]: already written this command at the bottom in the previous commit

File: docs/UserGuide.md [hunk 4]
```diff
@@ -230,7 +254,9 @@ Action     | Format, Examples
...
 **Find**   | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
+**Mark Task** | `mark task/TASK TITLE o/INDEX`<br> e.g., `mark task/Complete Project Proposal o/1`
```
Comment (yiwern5) [inline docs/UserGuide.md]: already written this command at the bottom in the previous commit

**Review by yiwern5 — APPROVED** [review]
LGTM

**Pull request merged**